api/Entity.go

```diff
@@ -221,3 +221,28 @@ func (vrsConnection *VRSConnection) GetAllEntities() ([]string, error) {
 
 	return uuids, nil
 }
+
+// EntityExists verifies if a specified entity exists in VRS
+func (vrsConnection *VRSConnection) EntityExists(uuid string) (bool, error) {
+    	readRowArgs := ovsdb.ReadRowArgs{
+        	Condition: []string{ovsdb.NuageVMTableColumnVMUUID, "==", uuid},
+        	Columns:   []string{ovsdb.NuageVMTableColumnVMUUID},
+    	}
+
+    	var uuidRows []map[string]interface{}
+    	var err error
+    	if uuidRows, err = vrsConnection.vmTable.ReadRows(vrsConnection.ovsdbClient, readRowArgs); err != nil {
+        	return false, fmt.Errorf("OVSDB read error %v", err)
+    	}
+
+    	var uuids []string
+    	for _, uuid := range uuidRows {
```

Comment:
Before this step; I will need to add a check for "nil" value of uuidRows interface above to handle the case wherein there is no UUID entry in VM Table